README.md

```diff
@@ -50,7 +62,11 @@ Lastly, the sample AzureSearchDotNetSample is a more detailed example that demon
 * Update the App.config with the service and api details of your Azure Search service
 * Compile and Run the project
 
-## Running the AzureSearchDotNetSample sample
+## Running the DotNetHowToSecurityTrimming sample
+
+See [Security filters for trimming Azure Search results using Active Directory identities](https://docs.microsoft.com/azure/search/search-security-trimming-for-azure-search-with-aad)
+

```

Comment:
I don't see any code changes for this in this PR.

Reply:
I had trouble getting the sample to run, so I deferred updating it for now.